docs/examples/PanelListGroupFill.js

```diff
@@ -0,0 +1,13 @@
+var panelInstance = (
+    <Panel header="Panel heading">
+      Some default panel content here.
+      <ListGroup fill>
+        <ListGroupItem>Item 1</ListGroupItem>
+        <ListGroupItem>Item 2</ListGroupItem>
+        <ListGroupItem>&hellip;</ListGroupItem>
+      </ListGroup>
+      Some more panel content here.
+    </Panel>
+  );
+
+React.render(panelInstance, mountNode);
```

Comment:
This example does not do anything see with and without fill.

![With Fill](http://cl.ly/image/3e0707451j0H/Image%202015-01-23%20at%202.37.40%20PM.png)

![Without Fill](http://cl.ly/image/1R2A3H122R3u/Image%202015-01-23%20at%202.38.48%20PM.png)